tests/torch/fx/test_models.py

```diff
@@ -156,3 +158,36 @@ def transform_fn(data_item):
 
         nncf_graph = GraphConverter.create_nncf_graph(quantized_model)
         check_graph(nncf_graph, get_dot_filename(model_case.model_id), FX_QUANTIZED_DIR_NAME)
+
+
+MODEL_COMRPESSION_MODES = [CompressWeightsMode.INT8_SYM, CompressWeightsMode.INT8_ASYM]
+
+
+TEST_MODELS_COMPRESSED = (
+    ModelCase(test_models.UNet, "unet", [1, 3, 224, 224]),
+    torchvision_model_case("resnet18", (1, 3, 224, 224)),
+    torchvision_model_case("mobilenet_v3_small", (1, 3, 224, 224)),
+    torchvision_model_case("vit_b_16", (1, 3, 224, 224)),
+    torchvision_model_case("swin_v2_s", (1, 3, 224, 224)),
+)
+
+
+@pytest.mark.parametrize("test_case", TEST_MODELS_COMPRESSED, ids=[m.model_id for m in TEST_MODELS_COMPRESSED])
+@pytest.mark.parametrize(
+    "compression_mode", MODEL_COMRPESSION_MODES, ids=[mode.name for mode in MODEL_COMRPESSION_MODES]
+)
+def test_compressed_model(test_case: ModelCase, compression_mode: CompressWeightsMode):
```

Comment:
Did you measure how long do these tests take?

Anyway, the goal of this test is unclear.

Reply:
I did not measure the time for it yet.
These tests are to check if the compressed model graph is generated correctly as per @daniil-lyakhov request (https://github.com/openvinotoolkit/nncf/pull/2891#discussion_r1723295655)

Reply:
@anzr299 thanks for the information. Please measure the time for these tests.

Reply:
It takes `66.12s` to run this test.

Reply:
Is it possible not to use reference graphs in this test? Compression changes only constants and it seems to me that it is possible to check the correctness of the changes in the test code.

Reply:
Alright, this test was mainly for the purpose of checking if the decompression nodes were placed correctly. I will remove the compression graph tests now.